lib/api_local.ml

```diff
@@ -69,6 +72,23 @@ let get_permalink ~ctx:_ ~(req : Slack_t.get_permalink_req) =
   printf "getting permalink for channel_id #%s and message_ts %s...\n" req.channel req.message_ts;
   Lwt.return_ok Slack_t.{ channel = req.channel; permalink = "SOME PERMALINK" }
 
+let get_upload_url_external ~ctx:_ ~(req : Slack_t.get_upload_url_ext_req) =
+  let json =
+    req |> Slack_j.string_of_get_upload_url_ext_req |> Yojson.Basic.from_string |> Yojson.Basic.pretty_to_string
+  in
+  printf "getting upload url for %s\n" req.filename;
+  printf "%s\n" json;
```

Comment:
isn't this debugging logic?

Are you adding tests to these functions? it seems that this logic should live in the test file?

(same below, for complete_upload_external)

Reply:
`api_local.ml` code is a module used for testing so the debugging code should live in here.